model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -155,6 +155,15 @@ message RegisterRequest {
 message RegisterResponse {
 }
 
+// One of the root applications specifying a bundle.
+// See https://s.apache.org/beam-breaking-fusion
+message Application {
```

Comment:
We shouldn't use the application, we should instead create a ProcessBundleDescriptor which explicitly connects the gRPC read port to the target, and send elements over the data plane.

Reply:
Agreed - removed this from ProcessBundleDescriptor. It's the runner's responsibility now to use the Application's in a BundleSplit to construct appropriate PBD's.